src/containers/nodes.js

```diff
@@ -26,9 +26,10 @@ class Nodes extends Component {
                                 </thead>
                                 <tbody>
                                     {this.props.nodes.map((node) => {
+                                        let shortID = node.ID.substring(0,8)
                                         return (
                                             <tr key={node.ID}>
-                                                <td><Link to={`/nodes/${node.ID}`}>{node.ID.substring(0,8)}</Link></td>
+                                                <td><Link to={`/nodes/${node.ID}`}>{shortID}</Link></td>
```

Comment:
This is done in multiple places (basically every time we get a UUID as ID). Could you change those as well to keep things consistent? Thanks!